src/main/webapp/add_item.xhtml

```diff
@@ -15,7 +15,7 @@
             <label>Vieta: </label>
             <h:inputText value="#{addItem.itemToCreate.location}" />
 
-            <br />
+            <br/>
```

Comment:
Just general note to the future - it is better to avoid br in styling. Add css with layout spacing.